### ISSUE TITLE
Strinova (1282270)

### ISSUE THREAD

**Issue by pollux78** [ValveSoftware/Proton#8262]
# Compatibility Report
- Name of the game with compatibility issues: Strinova
- Steam AppID of the game:1282270

## System Information
- GPU: RX 6700
- Video driver version: Mesa 24.2.7
- Kernel version: Kernel 6.11.9
- Link to full system information report : https://gist.github.com/polluxau/b0792a418e4da7f48e2c43d28795bed1
- Proton version: Experimental(bleeding edge)

## I confirm:
- [ ✅] that I haven't found an existing compatibility report for this game.
- [ ✅] that I have checked whether there are updates for my system available.

Launcher works great but the game itself doesnt launch, the launcher will either minimize itself and do nothing or the launcher will reappear after clicking play

## Reproduction
Click play on strinova, watch as the game never launches, Might be because of the anticheat being used (anticheat expert)

**Comment by Naitrate** [comment]
Interestingly enough, with Proton 9, the game runs on the Steam Deck without issues, but when I try on my laptop running NixOS, it goes absolutely nowhere outside the launcher.

**Comment by pollux78** [comment]
> Interestingly enough, with Proton 9, the game runs on the Steam Deck without issues, but when I try on my laptop running NixOS, it goes absolutely nowhere outside the launcher.

That's interesting as another game delta force was doing that also

Maybe valve gives the option to block desktop Linux users from accessing the game?

**Comment by Aemmie** [comment]
Devs also removed?? discussion about Steam Deck support, which is quiet suspicious. All positive reports from ProtonDB are from Steam Deck, while all negatives - from desktop users.
![image](https://github.com/user-attachments/assets/887872c0-63da-48a8-bb19-2822c412ec1a)

**Comment by MompfStar** [comment]
> Devs also removed?? discussion about Steam Deck support, which is quiet suspicious. All positive reports from ProtonDB are from Steam Deck, while all negatives - from desktop users. ![image](https://private-user-images.githubusercontent.com/10757826/389221387-887872c0-63da-48a8-bb19-2822c412ec1a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI0ODAzMDQsIm5iZiI6MTczMjQ4MDAwNCwicGF0aCI6Ii8xMDc1NzgyNi8zODkyMjEzODctODg3ODcyYzAtNjNkYS00OGE4LWJiMTktMjgyMmM0MTJlYzFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI0VDIwMjY0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlMGNhOTg1YmJiMjU4Y2JiNmI3MDM3MDgyYThhOTAwNjkyMzE0YTNiZmMxZDQxOGUxNzc1OTYzMDcxYWNiYzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jydBchAiKfPoK9oQyFWOVr57q5MWQwig0K0xBt5aXJo)

I am new to linux so please take my response with a grain of salt, but if the issue is whether or not the game is running off a steam deck or not would a possible fix be to to make proton think its running off a steam deck somehow?

**Comment by pollux78** [comment]
> > Devs also removed?? discussion about Steam Deck support, which is quiet suspicious. All positive reports from ProtonDB are from Steam Deck, while all negatives - from desktop users. ![image](https://private-user-images.githubusercontent.com/10757826/389221387-887872c0-63da-48a8-bb19-2822c412ec1a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI0ODAzMDQsIm5iZiI6MTczMjQ4MDAwNCwicGF0aCI6Ii8xMDc1NzgyNi8zODkyMjEzODctODg3ODcyYzAtNjNkYS00OGE4LWJiMTktMjgyMmM0MTJlYzFhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI0VDIwMjY0NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZlMGNhOTg1YmJiMjU4Y2JiNmI3MDM3MDgyYThhOTAwNjkyMzE0YTNiZmMxZDQxOGUxNzc1OTYzMDcxYWNiYzImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jydBchAiKfPoK9oQyFWOVr57q5MWQwig0K0xBt5aXJo)
> 
> I am new to linux so please take my response with a grain of salt, but if the issue is whether or not the game is running off a steam deck or not would a possible fix be to to make proton think its running off a steam deck somehow?

It is highly likely that you could spoof yourself for being a steamdeck but that could lead to a ban also probably 

I don't understand why these studios are choosing to support the steamdeck quietly and not desktop when the steamdeck is a desktop PC and you could do the same things as you could on a Linux desktop machine

**Comment by MompfStar** [comment]
Could it maybe be because the steam-deck has its own custom valve-kernel and there is something in there that allows the game to run?

**Comment by alosslessdev** [comment]
> Could it maybe be because the steam-deck has its own custom valve-kernel and there is something in there that allows the game to run?

It might be because of anticheat checking for the presence of a valve-compiled kernel

**Comment by eroc1990** [comment]
Based on [this comment](https://www.protondb.com/app/1282270#tIyHbu0Lqm) on ProtonDB, I'm inclined to believe it's a hardware check and not a kernel check. I don't believe Bazzite is using a Valve-compiled kernel, but this person was using Bazzite on a Steam Deck and the game did launch.

**Comment by pollux78** [comment]
> Based on [this comment](https://www.protondb.com/app/1282270#tIyHbu0Lqm) on ProtonDB, I'm inclined to believe it's a hardware check and not a kernel check. I don't believe Bazzite is using a Valve-compiled kernel, but this person was using Bazzite on a Steam Deck and the game did launch.

So technically what needs to happen is for desktops PC's to spoof themselves to being a steamdeck in some way so you can play the game, doesn't seem to matter what distro just the fact that your using a "steamdeck" allows you to play possibly

**Comment by Naitrate** [comment]
I think I tried using ``SteamDeck=1 %command%`` without much luck.

**Comment by 0x6A7232** [comment]
As of today (20250122), the launcher launches, if you click play the launcher disappears, then nothing happens and the launcher reappears as if you had opened the game and closed it.  Tried on GE Proton 9-23, Proton Hotfix, Proton Experimental, and Proton 9.0-4

**Comment by ejrydhfs** [comment]
> As of today (20250122), the launcher launches, if you click play the launcher disappears, then nothing happens and the launcher reappears as if you had opened the game and closed it. Tried on GE Proton 9-23, Proton Hotfix, Proton Experimental, and Proton 9.0-4

This sounds like anticheat checking for a real windows install

**Comment by eroc1990** [comment]
That doesn't explain why it seems to have worked on Steam Deck (though there don't seem to be any recent reports corroborating playability on the Deck recently).

**Comment by 0x6A7232** [comment]
> That doesn't explain why it seems to have worked on Steam Deck (though there don't seem to be any recent reports corroborating playability on the Deck recently).

Could have been as simple as them accidentally pushing a version without anti-cheat enabled (perhaps Early Access?) then that changing later.  Who knows.  All I know is that currently it's broken.

**Comment by Naitrate** [comment]
> > That doesn't explain why it seems to have worked on Steam Deck (though there don't seem to be any recent reports corroborating playability on the Deck recently).
> 
> Could have been as simple as them accidentally pushing a version without anti-cheat enabled (perhaps Early Access?) then that changing later. Who knows. All I know is that currently it's broken.

From what I see on SteamDB, there's two depots, one with the game content, the other with the launcher itself. There's no indication that this is occurring.

**Comment by xiaguangbo** [comment]
fedora 41 x86_64 have same problem

**Comment by Ryoxzd** [comment]
Is there still no solution?

**Comment by Naitrate** [comment]
Allegedly, Mecha Break's new beta is using an up to date version of ACE (Anticheat Expert) which only works on Steam Deck. Strinova seems to be using the same anti-cheat, and the same for Infinity Nikki (Which apparently recently broke). So the anti-cheat might be the reason why it won't work outside of the Steam Deck.

[https://www.gamingonlinux.com/2025/02/anti-cheat-stops-mecha-break-running-on-desktop-linux-but-works-on-steam-deck/](https://www.gamingonlinux.com/2025/02/anti-cheat-stops-mecha-break-running-on-desktop-linux-but-works-on-steam-deck/)

Allegedly, the anti-cheat lets the game work on Bazzite, which makes me wonder if there's some sort of hardware check being done (like for the CPU).

**Comment by 0x6A7232** [comment]
Going to try this:

https://www.youtube.com/watch?v=i_Ef9VCcVSw 

Go to your Steam **LIBRARY** search Proton then select Proton EasyAntiCheat Runtime and INSTALL. Then right-click Strinova, go to Compatibility and tell it to run using the Proton EasyAntiCheat Runtime you just installed.  Don't know if this will work but I'll give it a shot.

**Comment by 0x6A7232** [comment]
> Going to try this:
> 
> https://www.youtube.com/watch?v=i_Ef9VCcVSw
> 
> Go to your Steam **LIBRARY** search Proton then select Proton EasyAntiCheat Runtime and INSTALL. Then right-click Strinova, go to Compatibility and tell it to run using the Proton EasyAntiCheat Runtime you just installed. Don't know if this will work but I'll give it a shot.

I was apparently wrong about Proton EasyAntiCheat Runtime: it isn't a compatibility tool you select, it is something any compatibility tool can use if EAC is supported by the game.
I read some noise about using `PROTON_EAC_RUNTIME=$HOME/.steam/steam/steamapps/common/Proton\ EasyAntiCheat\ Runtime/ %command% ` in the launch options for the game but that doesn't seem to be an actual path (there is Proton x.0 but no Proton folder, and none have an EasyAntiCheat folder, in fact, I cannot seem to find an EasyAntiCheat folder on my system with a brief search) so I doubt it will work (EDIT: it didn't work).  Back to square 1 I guess.

**Comment by Ryoxzd** [comment]
the good news is they are probably trying to fix this,https://steamcommunity.com/app/1282270/discussions/4/4626981087516566932/

**Comment by Ryoxzd** [comment]
[demo video](https://files.catbox.moe/mqojp0.mp4)
I tested this using Wine 10.7 (Staging) — the game runs, but feels a bit unstable.
I'm not entirely sure if the instability is due to recording the screen at the same time or just the limitations of my hardware.
Further testing without recording might help clarify the cause. Sometimes the game will crash i'm trying to figure out how to make it work better.

**Comment by MompfStar** [comment]
> [demo video](https://files.catbox.moe/mqojp0.mp4) I tested this using Wine 10.7 (Staging) — the game runs, but feels a bit unstable. I'm not entirely sure if the instability is due to recording the screen at the same time or just the limitations of my hardware. Further testing without recording might help clarify the cause. Sometimes the game will crash i'm trying to figure out how to make it work better.

Have you done anything else for the game to run? As I have tried to follow the steps you have and am able to play the game for a few minutes before it crashes due to an anti-cheat error


Edit:
The Anticheat that shows up is ACE

**Comment by Ryoxzd** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2883611726

Hey! Thanks for pointing that out — turns out the issue happened because winedbg was triggered.
I fixed it by opening regedit via Wine in your prefix and navigating to:

`
HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\AeDebug`

Then I set Auto to 0 and cleared the Debugger key. That prevents winedbg from launching when a crash happens.
Appreciate the heads-up!

**Comment by MompfStar** [comment]
Just tried with removing the wine debugger and the game runs quite well, I get around 200FPS and the crashing has stopped

**Comment by ktnk-dev** [comment]
![Image](https://github.com/user-attachments/assets/696d40d7-9e02-401b-a281-a11012503882)
Running using Lutris on Fedora 42. Everything works perfectly fine. Sometimes textures missing in dormitory, but enabling all graphics features in Lutris (DXVK, VKD3D, etc) fixes this

![Image](https://github.com/user-attachments/assets/df278b91-caf5-43b5-b7c8-9e39557d98b3)
Always appears this error, you can ignore it

**NON**-Steam version. @Ryoxzd, thank you for [this comment](https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2883922996)

Overall experience: Runs mostly perfectly, with occasional lag spikes (never happened on windows before)

**Comment by zahmbie1** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2887766207

Can confirm that this works, but running this with two monitors requires that you either: turn on fullscreen, which turns off the secondary monitor, or turn on windowed mode. Borderless fullscreen makes the game stretch across both monitors.
Running Linux Mint 22.1
EDIT: This appears to be a Wine 10.7 issue, not an issue with the game itself. I tried GFL2 and it had the same issue.

**Comment by 0x6A7232** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2883922996

Wouldn't just setting Auto to 0 work?  As then if you needed to use it you could just put it back to 1.  I *think* setting Auto to 0 will work.  Can anyone test?

For the record, the String Value (REG_SZ) "Debugger" default is: `winedbg --auto %ld %ld`
(So if you want to re-create it, right-click > New > String Value then double-click and enter the above value)

**Comment by ktnk-dev** [comment]
@zahmbie1 
> Can confirm that this works, but running this with two monitors requires that you either: turn on fullscreen, which turns off the secondary monitor, or turn on windowed mode. Borderless fullscreen makes the game stretch across both monitors.

Strinova runs normaly on my setup with two monitors (Hyprland, Wayland, Wine 10.7, tested with Fullscreen, Borderless and Windowed mode). I think its DE issue, not an issue with wine

EDIT 1: tested few days ago before game update
EDIT 2: tested after game update - no issues

**Comment by ktnk-dev** [comment]
I have problems with updating game launcher (and game itself) on Wine 10.7. Launcher crashes when downloading files. 

`GE-Proton (Latest)` with disabled DXVK and dgvoodoo2 helps me 

![Image](https://github.com/user-attachments/assets/e8c4b678-4b93-4d0d-b31b-a47294fa7e65)

**Comment by ktnk-dev** [comment]
@0x6A7232 
> Wouldn't just setting Auto to 0 work? As then if you needed to use it you could just put it back to 1. I think setting Auto to 0 will work. Can anyone test?

Game crashes with AC error 

![Image](https://github.com/user-attachments/assets/1e8edb01-6793-4477-81db-fd6cb4900eef)

![Image](https://github.com/user-attachments/assets/cb47e3b6-800c-4e4a-b671-3ff47717a697)

Removing "Debugger" value (making it empty instead deleting) removes error and annoying console windows

**Comment by 0x6A7232** [comment]
> [@0x6A7232](https://github.com/0x6A7232)
> 
> > Wouldn't just setting Auto to 0 work? As then if you needed to use it you could just put it back to 1. I think setting Auto to 0 will work. Can anyone test?
> 
> Game crashes with AC error
> 
> <snip>
> 
> <snip>
> 
> Removing "Debugger" value (making it empty instead deleting) removes error and annoying console windows

So, for the fix to work, you must both set Auto to 0 and remove the `winedbg --auto %ld %ld` value from Debugger?  Odd.  I wonder why that is.  Let me research that a bit and get back, hmm.

**Comment by 0x6A7232** [comment]
Ok, did some AI research, [link](https://x.com/i/grok/share/Go04fSF7a4EyIxhQuara6UjOH) (skip past first prompt and response, I had forgotten to include this thread as a reference).

My tl;dr guess from the below info: Anti-Cheat Expert simulates, or at the least, inspects, the debug entry for Strinova, and, upon seeing the custom `winedbg` entry, trips it off somehow, or if it doesn't, keeps close tabs on it, causing lag.

<details><summary>It appears (well, the AI thinks):</summary>

> The registry key `HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\AeDebug` in Windows is used to configure the system's **Application Error Debugging** behavior, specifically for handling application crashes. Here's a breakdown of its role, how WINE interacts with it, and why the described changes in the Strinova thread might be necessary:
> 
> ### 1. Purpose of the AeDebug Registry Key in Windows
> The `AeDebug` key controls how Windows responds when an application crashes (e.g., encounters an unhandled exception). It determines whether a debugger is automatically invoked and how it behaves. The key typically contains two important values:
> 
> - `Auto` **(REG_SZ)**: 
>   - When set to `1`, Windows automatically launches the debugger specified in the `Debugger` value without prompting the user when an application crashes.
>    
>   - When set to `0`, Windows prompts the user (e.g., with a dialog box) before launching the debugger, allowing manual intervention or cancellation.
> 
> - `Debugger` **(REG_SZ)**: Specifies the command line for the debugger to be launched when an application crashes. For example, it might point to `drwtsn32.exe` (Dr. Watson, a legacy Windows debugger) or another debugger like `winedbg` in WINE's case.
> 
> Normally, this key is used by Windows to facilitate debugging of crashed applications, either by developers or system administrators, to capture crash dumps or analyze the cause of the failure.
> 
> ### 2. WINE's Interaction with AeDebug
> WINE, the compatibility layer used by Proton to run Windows applications on Linux, emulates much of the Windows environment, including the Windows registry. When a Windows application runs under WINE, WINE sets up a **WINE prefix** (a virtual Windows environment) that includes a registry mimicking the Windows registry. In this prefix, WINE configures the `AeDebug` key to use `winedbg`, its built-in debugger, to handle application crashes.
> - By default, WINE sets:
>   - `Auto` to `1`, meaning `winedbg` is automatically invoked when an application crashes.
> 
>   - `Debugger` to something like `winedbg --auto %ld %ld`, where `%ld %ld` are placeholders for process ID and thread ID, allowing `winedbg` to attach to the crashed process.
> 
> When an application (like Strinova) crashes, WINE's `winedbg` attempts to handle the crash, which may involve generating a backtrace or crash log. However, this behavior can interfere with some applications, especially those with anti-cheat systems or custom crash-handling mechanisms, leading to issues like the game failing to launch or hanging.
> 
> ### 3. Context from the Strinova Thread
> The GitHub issue for Strinova reports that the game’s launcher works, but the game itself fails to launch, potentially due to the anti-cheat system (Anticheat Expert). The proposed solution involves using `protontricks` to modify the WINE prefix’s registry:
> 
> - Set `Auto` to `0` to disable automatic debugger invocation.
> 
> - Clear the `Debugger` value (erase `winedbg --auto %ld %ld`) to prevent WINE from launching `winedbg` at all.
> 
> This suggests that `winedbg` is interfering with Strinova’s operation, likely because the anti-cheat system detects the debugger’s presence or behavior as suspicious (a common issue with anti-cheat software, which often blocks debuggers to prevent cheating or reverse engineering).
> 
> ### 4. Why Does This Cause Problems?
> Even with `Auto` set to `0`, issues may persist because:
> - **Anti-Cheat Sensitivity**: Anti-cheat systems like Anticheat Expert are designed to detect and block any debugging activity, as debuggers can be used to manipulate game memory or bypass security. Even if `Auto` is `0`, the presence of a `Debugger` value (pointing to `winedbg`) might still trigger anti-cheat checks, as some anti-cheat systems proactively scan the registry or environment for debugging configurations.
> 
> - **WINE’s Crash Handling**: WINE may still attempt to handle crashes in a way that interacts with the `AeDebug` settings, even if `Auto` is `0`. For example, if the application crashes and WINE checks the `Debugger` value, it might attempt to launch `winedbg` or behave unexpectedly, causing a hang or failure.
> 
> - **Strinova’s Anti-Cheat Behavior**: The thread suggests that Strinova’s anti-cheat (Anticheat Expert) may be particularly sensitive. When the launcher attempts to start the game, the anti-cheat could detect `winedbg`’s potential involvement (via the registry) and block the game from launching, either by minimizing the launcher or causing it to reappear without starting the game.
> 
> ### 5. Why Does `Auto = 1` Cause Problems Even Without Debug Mode?
> The `Auto` value being set to `1` means `winedbg` is automatically invoked on any crash, which can cause issues even if the game isn’t explicitly run in debug mode:
> 
> - **Anti-Cheat Interference**: As mentioned, anti-cheat systems often block any debugger, including `winedbg`, to prevent tampering. If Strinova crashes (or the anti-cheat simulates a crash to test the environment), `winedbg`’s automatic invocation could be flagged as suspicious, causing the game to fail.
> 
> - **Resource Conflicts or Hangs**: When `winedbg` is launched, it may consume resources or interfere with the game’s process, leading to hangs or crashes. For example, `winedbg` might attempt to pause the process for debugging, which could conflict with the game’s or anti-cheat’s expected behavior.
> 
> - **Unexpected Crashes**: Even in normal operation, minor errors in the game or WINE’s emulation might trigger a crash that invokes `winedbg`. This could happen without the user explicitly enabling debug mode, as WINE’s default configuration assumes debugging is enabled for crash handling.
> 
> ### 6. Why Clearing `Debugger` and Setting `Auto` to `0` Helps
> - **Disabling `winedbg` Completely**: By clearing the `Debugger` value, WINE no longer has a debugger to invoke, effectively disabling WINE’s crash-handling mechanism. This prevents `winedbg` from interfering with the game or being detected by the anti-cheat.
> 
> - **Preventing Automatic Debugging**: Setting `Auto` to `0` ensures that even if a debugger were specified, WINE wouldn’t automatically launch it, reducing the chance of anti-cheat detection or resource conflicts.
> 
> - **Anti-Cheat Compatibility**: Many anti-cheat systems are less likely to flag an environment where no debugger is configured, allowing Strinova to launch successfully.
> 
> ### 7. Additional Context from the Thread
> The Strinova issue (#8262) indicates:
> 
> - The game uses **Anticheat Expert**, which is known to be strict and may block environments with debugging tools.
> 
> - The launcher works, but the game fails to launch, suggesting the anti-cheat kicks in during the transition from launcher to game executable.
> 
> - The solution (modifying `AeDebug`) was proposed in a [comment](https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2450684967), confirming that disabling `winedbg` via the registry resolved the issue for at least one user.

</details>

(There's more, but it's extraneous, you can check the prompt thread and follow-up if you'd like)

**Comment by 0x6A7232** [comment]
Side note: Anyone know what Strinova's launcher uses to play background music?  As that's not working.  I can change the volume, but nothing is playing (sound / music works fine in other games and their launchers).  In the video tab, the video plays but there's no audio.  I do see:
```
/SteamLibrary/steamapps/common/
  Strinova/Launcher/audio/
    qtaudio_wasapi.dll
    qtaudio_windows.dll
  Strinova/Strinova/Engine/Binaries/ThirdParty/Ogg/Win64/VS2015/
    libogg_64.dll
```
-- but libogg_64.dll is probably for the game's internal use and the Launcher uses the stuff in /Strinova/Launcher

Anyone else have missing BGM or is it just me?  Just tried uninstall / reinstall and it didn't fix it.

Also, I still get the OP's error where the launcher disappears upon clicking Play, then re-appears and the game doesn't launch.  Tried Proton Experimental, Proton Hotfix, and GE-Proton 9-27

**Comment by Ryoxzd** [comment]
> Replying to [#8262 (comment)](https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2906765863)

Hi, I believe I’ve identified the reason why the BGM (background music) isn't playing in the Strinova Launcher on Linux.

The launcher appears to use Qt Multimedia, which depends on GStreamer for audio/video playback on Linux. Interestingly, when running the launcher with Proton 8, the audio plays correctly—likely because GStreamer is still included or better supported in that version. However, newer versions of Proton (e.g., Proton 9 or GE-Proton) seem to lack this support by default.

**Comment by ktnk-dev** [comment]
@0x6A7232 
> Side note: Anyone know what Strinova's launcher uses to play background music? 

wait what? I never thought launcher had bg music (maybe because I always mute it)

**Comment by ktnk-dev** [comment]
@0x6A7232 
> My tl;dr guess from the below info: Anti-Cheat Expert simulates, or at the least, inspects, the debug entry for Strinova, and, upon seeing the custom winedbg entry, trips it off somehow, or if it doesn't, keeps close tabs on it, causing lag

I think this is just yet another anti-debug thing that works in every system

btw yes - setting Auto to 0 forces this(image) prompt to appear 
![Image](https://github.com/user-attachments/assets/e940b1af-754f-481b-89af-8e1881384c9e)

but if you press any of the buttons (yes, no) game just crashes. you should simply **ignore** this error and continue play normally. I think with Auto 1 game will just crash even with empty Debugger value

**Comment by 0x6A7232** [comment]
> > Replying to [#8262 (comment)](https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2906765863)
> 
> Hi, I believe I’ve identified the reason why the BGM (background music) isn't playing in the Strinova Launcher on Linux.
> 
> The launcher appears to use Qt Multimedia, which depends on GStreamer for audio/video playback on Linux. Interestingly, when running the launcher with Proton 8, the audio plays correctly—likely because GStreamer is still included or better supported in that version. However, newer versions of Proton (e.g., Proton 9 or GE-Proton) seem to lack this support by default.

I tested with Proton 8 and it just says "Please use Steam Client to launch this game" and exits.  You sure it was 8.0?   I also tried 9.0 but it behaves the same as Experimental and GE-Proton 9-27  EDIT: 7.0 and 6.3 fail the same way as 8.0 unsurprisingly, and I assume all older versions will as well. If someone wants me to check a specific version let me know, though.
EDIT2: Installed GE-Proton 10-3, it behaves exactly the same as 9-27 (no BGM, launch game button press = launcher restart).

**Comment by Ryoxzd** [comment]
> I tested with Proton 8 and it just says "Please use Steam Client to launch this game" and exits. You sure it was 8.0? I also tried 9.0 but it behaves the same as Experimental and GE-Proton 9-27 EDIT: 7.0 and 6.3 fail the same way as 8.0 unsurprisingly, and I assume all older versions will as well. If someone wants me to check a specific version let me know, though. EDIT2: Installed GE-Proton 10-3, it behaves exactly the same as 9-27 (no BGM, launch game button press = launcher restart).

Yeah, I'm currently using Proton 8.0-5, and only the launcher can be opened with it — along with the BGM playing fine. However, the game itself isn't fully playable through Proton.

![Image](https://github.com/user-attachments/assets/22403341-d5fc-47cc-b9d5-e86cf54e07bc)

**Comment by 0x6A7232** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2909085650

I don't even get that far.  No music, launcher won't load, just the error message (Strinova themed so it's from the launcher but the main launcher window doesn't show) and then you click to exit.

**Comment by ghost** [comment]
i got it to launch, i played some time and it crashed. now i can not start it again due to this error in lutris:

```
err:   Failed to create shared resource: VK_KHR_EXTERNAL_MEMORY_WIN32 not supported
[284:288:0604/020749.736:ERROR:network_change_notifier_win.cc(142)] WSALookupServiceBegin failed with: 8

```

**Comment by ghost** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2887766207

i am also on fedora. i got it to run once then it crashes and wont open anymore. see me other comment. im getting some wierd unhandled exeption at xblabla error

![Image](https://github.com/user-attachments/assets/da065e18-f17b-496b-9871-5f6d6a517415)


**EDIT1**
i think i fixed it!!!! it was due ntsync enabled wine, so i diabled esync and fsync, and on fedora did a `sudo modprobe ntsync`

**EDIT2**
it shows 

![Image](https://github.com/user-attachments/assets/dce37e51-0d35-4c49-8bba-935715c0416a)

some times, choosing any option and game crashes. but the problem is that it is stealing focus of the game....

**EDIT 3**
just played tutorial, did not crashed. i simply enabled full screen exclusive in settings. 

![Image](https://github.com/user-attachments/assets/d3ea250d-cae7-4f23-a7d4-793b7d4e9473)

As for the wine used: 
- https://github.com/future-XD/wine-tkg-staging-ntsync-git
- load it in lutris by placing it in the runners map
- disable esync, fsync.
- make sure ur kernel has ntsync support and it is loaded!

the anoying pop up still comes up but due to fullscreen it does not pop over the game. so now the game is perfectly playable! Buttery smooth on my specs, ntsync works.... i would even consider it runs more smooth on linux then windows

**Comment by 0x6A7232** [comment]
I just updated my system and I'm now on Wine-Staging 10.9, MX Linux 23.6, kernel 6.14.9-1-liquorix-amd64 (Liquorix is usually bleeding edge).  Also updated to GE-Proton 10-4 
(PSA, 10-4 now includes FSR4 support I guess? Look at the release notes if you want to try that, I think you need `FSR4_UPGRADE=1` in the launch options but I didn't try that here).

How did you disable esync and fsync, if I can pass that in the Steam options it might work for me.

I tried using `PROTON_NO_ESYNC=1 PROTON_NO_FSYNC=1 PROTON_USE_NTSYNC=1 %command%` in the Steam launch options, but the behavior is the same as before.  Opened a terminal and `lsof /dev/ntsync` shows nothing, not with the menu and not when attempting to launch the game from the menu and not when the menu re-appears.  (But it does exist; I verified with `ls /dev/ntsync`)

EDIT: Bit of an update, this is apparently how you can check to see if you have ntsync enabled and force it to load (doesn't work for me but maybe it will help someone out):

`uname -r`
**(check running kernel version)**
`6.14.9-1-liquorix-amd64`
**(your running kernel version)**

`grep -i NTSYNC /boot/config-`
**(checking for ntsync enable in kernel compile config; press Tab a few times to bring up list : )**
```
config-6.11.9-amd64             config-6.14.6-1-liquorix-amd64  config-6.14.9-1-liquorix-amd64
config-6.12.17-amd64            config-6.14.7-1-liquorix-amd64  
config-6.14.5-1-liquorix-amd64  config-6.14.8-1-liquorix-amd64 
```

`grep -i NTSYNC /boot/config-6.14.9-1-liquorix-amd64`
**(type enough to select your kernel then Tab to finish, or type it all out, press enter to check ntsync support)**
`CONFIG_NTSYNC=y`
**(preferably, you want this result to be y, not n - if it is y, the kernel was compiled with built-in ntsync,)**
**(if your result is n, keep going to check for ntsync support by loadable kernel module)**

`depmod -av | grep ntsync`
**OR**
`modinfo ntsync`
**(use one or both of those two to check for the ntsync kernel module)**

`lsmod | grep -i ntsync`
**(check if the ntsync kernel module is running)**

`sudo modprobe ntsync`
**(if not, load ntsync kernel module)**

`ls /dev/ntsync`
**(checking if it exists)**
`/dev/ntsync`
**(yup)**

`lsmod | grep -i ntsync`
**(check if the ntsync kernel module is running now that it has explicitly been enabled)**
`ntsync                 20480  0`
**(this the result you want)**

`lsof /dev/ntsync`
**(check if anything is running using ntsync; do this when a game / app is supposed to be using it)**

`ls -l /etc/modules`
**(check if this file is used by your system to load kernel modules on boot)**
`ls -l /etc/modules-load.d`
**(check if these .conf files are used by your system to load kernel modules on boot)**

To load the ntsync kernel module on every boot, either add `ntsync` to the list of modules in the /etc/modules file, or make a new file called `ntsync.conf` in the /etc/modules-load.d directory and insert the word `ntsync` into the new ntsync.conf file and save, depending on which method your system uses.

**Comment by ghost** [comment]
> usually bleeding edge). Also updated to GE-Proton 10-4

1. u need lutris / bottles. i could not get the steam version working
2. Geproton is for steam. use ge wine / tkg wine, or the one i gave
3. to disable esync and fsync is only needed if u plan to use ntsync enabled wine. i did, so i disabled it in the lutris options
4. ikd if ur kernel comes with ntsync patches. if it does not, no point in using wine with ntsync enabled.

**Comment by 0x6A7232** [comment]
See edit above.  I'm troubleshooting.  Possible even GE-Proton 10-4 doesn't use ntsync, but I find that odd.  You'd think it would be supported but would need to be enabled with `PROTON_USE_NTSYNC=1 %command%` but what do I know lol.

**Comment by ghost** [comment]
> See edit above. I'm troubleshooting. Possible even GE-Proton 10-4 doesn't use ntsync, but I find that odd. You'd think it would be supported but would need to be enabled with `PROTON_USE_NTSYNC=1 %command%` but what do I know lol.

u need to use stand alone launcher, not with proton ge since that is for steam but the wine version

**Comment by ghost** [comment]
for those intrested i made a youtube guide sry for bad english

https://www.youtube.com/watch?v=ylbCdFw0ACU

**Comment by ghost** [comment]
https://github.com/aamaanaa/strinova-linux

i made a repo with the guide in it and most important, a lutris installer script that automates it all.

**Comment by 0x6A7232** [comment]
> > See edit above. I'm troubleshooting. Possible even GE-Proton 10-4 doesn't use ntsync, but I find that odd. You'd think it would be supported but would need to be enabled with `PROTON_USE_NTSYNC=1 %command%` but what do I know lol.
> 
> u need to use stand alone launcher, not with proton ge since that is for steam but the wine version

Just to confirm, this will work on the Steam version of Strinova and launch it outside of Steam, or do I need to install a non-Steam version of Strinova?



> https://github.com/aamaanaa/strinova-linux
> 
> i made a repo with the guide in it and most important, a lutris installer script that automates it all.

Bless you.  I'm not sure how, but I know you can get install scripts to be searchable by Lutris, I'm not sure ifyou have already done that it but if you do,  it would make this automatic for most people. 



> for those intrested i made a youtube guide sry for bad english
> 
> https://www.youtube.com/watch?v=ylbCdFw0ACU

No problems, thank you for creating a guide in your second language!

**Comment by ghost** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2941946028

no, it wil not work on steam version. u need the stand alone version in lutris so outside of steam. Steam has nothing to do with it then, wich is what we want.

As for the lutris script:

![Image](https://github.com/user-attachments/assets/226ec07f-23d7-465b-a91f-c4ae8911fea8)

download the yaml file from the repo i sent u, and in lutris u select "install from a local install script". that is it and the launcher will start. 

i submitted the script to lutris repo, so it is aviable to anyone, but a moderator must check it first, so that will take some time hence you need to manualy load in the file

For futher trouble, or trouble shooting:

- either see the youtube video, AND the description of it
- open github issue on my repo, if u really can not get it to work AFTER u have done everything in the description in my own repo.

**Comment by Ryoxzd** [comment]
> Replying  to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2942165676

Hi! Thank you for the detailed explanation and the script.

I was wondering—does this method also work for other games that use ACE anti-cheat? I'm really curious if the same approach with the standalone version through Lutris can be applied to other ACE-protected titles as well.

Looking forward to your response!

**Comment by ktnk-dev** [comment]
@aamaanaa https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2937861738
> some times, choosing any option and game crashes. but the problem is that it is stealing focus of the game.... 

Yes, but this happens only after fixed amount of time, so i just wait this error to appear and Alt+Tab back to game

**Comment by ktnk-dev** [comment]
@aamaanaa https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2937861738
> disable esync, fsync.

I dont disabled them and all works fine. maybe gnome (other DE) related issue?

**Comment by ktnk-dev** [comment]
@aamaanaa https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2942165676
> download the yaml file from the repo i sent u, and in lutris u select "install from a local install script". that is it and the launcher will start.

I used "Install a Windows game from an executable" option, selected launcher setup executable and then manually configure lutris to run launcher executable 

EDIT: i checked your yaml, seems good, but cant test it because of poor network (cant afford to reinstall whole game)
also it may crash launcher during update, because for installing update you need select ProtonGE as runner, wine dont work for some reasons (https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2902543011)

**Comment by wjpattyn** [comment]
The Lutris solution seems to work great for me. Thanks for sharing. I have recently made the switch to Linux, but for Strinova I still had to dual boot into Windows up until now.

Only issue I really have is that the models in the menus look off. It's like they're missing proper lighting and the edges are also visible. This is also apparent in the character select screen before a match. The map in the background in the character select screen also loads partially. In the match itself, the models and the map look proper though. Running on everything on ultra settings.

Just wanted to share my experience but if anyone has an idea how to resolve it, that would be great. The game is perfectly playable otherwise.

![Image](https://github.com/user-attachments/assets/2d85b89a-0ab7-458d-a87f-0b732e286cd7)

<details>
  <summary>Click for more screenshots</summary>

Background
![Image](https://github.com/user-attachments/assets/e713db5d-733a-4498-a0e4-3690b7da0e74)

Weapon + character during character select
![Image](https://github.com/user-attachments/assets/e52c2f6e-6fca-4e9c-9d84-017eddb62253)

During match (looks fine)
![Image](https://github.com/user-attachments/assets/a673da3a-4013-4a0d-a5cd-9f0f85a8e748)

After match (looks fine in this case). Although I had instances were the lighting was also off. Edges are visible though and the mask she's holding has some weird stuff going on.
![Image](https://github.com/user-attachments/assets/044066d4-9e1a-4d3c-a68f-0a91c8a81b5d)
  
</details>

![Image](https://github.com/user-attachments/assets/d86d4ce9-9197-4354-8627-2ad2a72a38d8)

**Comment by ghost** [comment]
> [@aamaanaa](https://github.com/aamaanaa) [#8262 (comment)](https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2937861738)
> 
> > disable esync, fsync.
> 
> I dont disabled them and all works fine. maybe gnome (other DE) related issue?

i use fedora gnome with that kernel having ntsync, wich i enabled with mod probe. ntsync did not worked if i did not turn off esync and fsync, so idk if u use ntsync kernel and ntsync wine, becuase if u do u cant have esync or fsync enabled.

**Comment by ghost** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2946365651

manual method works just fine. i added the yaml for automating a large part for those who install it first time.

**Comment by aamaanaa** [comment]
> Strinova

models look off. i did not have such issues. idk how this can be.

**Comment by ghost** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2953691047

models look off but i did not have such an issue on my side. maybe because of hardware, distro used idk.

**Comment by ktnk-dev** [comment]
@wjpattyn  https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2953691047
same issue with missing textures on my first installation. fixed it by trying another DXVK versions
also try wine 10.7 instead 10.9 (no game reinstall required, just downgrade it using your package manager)

EDIT: i use amdgpu, you can also try change something in nvidia settings, this also can help, but i sure it is wrong dxvk version

**Comment by ktnk-dev** [comment]
@aamaanaa https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2953943856
> manual method works just fine. i added the yaml for automating a large part for those who install it first time.

yea, you did good job automating it, just saying for ppl with game/launcher updating issue in future.

**Comment by mrvictory1** [comment]
The launcher doesn't have an "import game" function, here are the manual steps:
* Copy game to drive_c/Strinova/Game
* In drive_c/Strinova/config.ini change `CurrentGameVersion` to `1.3.10.21` and `CurrentStage` to `INITSTAGE`. The version will fall out of date over time, copy new value from Windows or start downloading fresh to get current version string.

**Comment by wjpattyn** [comment]
@ktnk-dev https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2954237872

Thanks for the reply. Unfortunately I am out of luck. I've tried downgrading to wine 10.7 and fiddling with different DXVK versions (2.2, 2.3, 2.4, 2.6.1), switched to proton Vulkan.

The NVIDIA driver I'm on is 570 (open kernel). Maybe downgrading could affect something, but I rely on this version for other applications to work. So I won't downgrade for now. But other people facing the same issue could try. I might fiddle some more with different wine versions and try some other combinations later.

During a match, the textures look good so all in all I can live with it. Perhaps I'll consider an AMD GPU in the future whenever I need to upgrade 🙃

**Comment by ktnk-dev** [comment]
@wjpattyn https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2955142346

well at least you still can play relatively comfortably. I also want to upgrade my gpu to Nvidia one in a few months, so I also will test this myself 

also I don't think there is any wine supposed below 10.7. how did you manage to run strinova on proton? it don't work for me

**Comment by ktnk-dev** [comment]
@mrvictory1 https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2954780924

what do you mean? you tried setup and existing installation? I never seen this before

**Comment by mrvictory1** [comment]
@ktnk-dev these steps are for importing game from Windows if you already have it installed, for downloading the steps are not needed.

**Comment by wjpattyn** [comment]
@ktnk-dev https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2955188213

I worded it incorrectly. I meant VKD3D-Proton.

Now that you've mentioned it, I just tried running it via Proton/Proton-GE since I didn't do that before. But that indeed doesn't run.

**Comment by ktnk-dev** [comment]
@wjpattyn https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2955313856
> But that indeed doesn't run.

as excepted :)

**Comment by ghost** [comment]
Some good news. the lutris team has published my yaml lutris installer, altough with some changes:

![Image](https://github.com/user-attachments/assets/0f7539f2-3bcd-4779-a58f-e8a35c86bae7)

![Image](https://github.com/user-attachments/assets/ed16015d-4687-42d9-b375-5e4f07ced59e)

they removed the link to my git repo so i have no way of knowing if it breaks for some....


lets hope the download url does not break.
also, the git will always have the latest version. this is becuase lutris mods take quite a long time to revieuw and submit.

they added

`wine:
  overrides:
    tdh.dll: n`

so maybe that has something to do with that exception window idk

**Comment by ghost** [comment]
> > Replying  to [#8262 (comment)](https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2942165676)
> 
> Hi! Thank you for the detailed explanation and the script.
> 
> I was wondering—does this method also work for other games that use ACE anti-cheat? I'm really curious if the same approach with the standalone version through Lutris can be applied to other ACE-protected titles as well.
> 
> Looking forward to your response!

some one told me they got infinity nikki to work. but i tried it myself and could not get around some kind of weird anti cheat files validation error.

p.s sry for late reply

**Comment by mrvictory1** [comment]
that was me :)
recheck the comment, I made a mistake writing the steps and edited it now.

**Comment by wjpattyn** [comment]
https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2955475818
> 
> they added
> 
> `wine: overrides: tdh.dll: n`
> 
> so maybe that has something to do with that exception window idk

This does seem to prevent the exception window from appearing. I'm having the game open for a while, played a match and it doesn't appear anymore. Good stuff.

**Comment by ghost** [comment]
> wine: overrides: tdh.dll: n

nice! ill update it in my own github repo

**Comment by ghost** [comment]
> that was me :) recheck the comment, I made a mistake writing the steps and edited it now.

i managed to get infinity nikki to run WITOUT modifying game files!!!

https://imgur.com/a/DcQuvWa

**Comment by ghost** [comment]
why is my comments (https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2955475818) marked as off topic? if i did not submit the script to the lutris team, we would not have know of  the `tdh.dll: n` bypass to prevent the exception window in strinova.... it seems to me to be very relevant information. 

i would like to have a explanation from whomever who did that.


EDIT: thanks. ill see if i can make seperate issue / repo of my own for other games. let us keep this to strinova only.

**Comment by mrvictory1** [comment]
create a seperate issue for infinity nikki. Also can you write your installation steps? I could not reproduce your method.

**Comment by 0x6A7232** [comment]
Do you get BGM music in the menus?  Just installed using the new Lutris script (thx again @aamaanaa) and still don't have that.

<details> <summary> > The Lutris solution seems to work great for me. </summary> > The Lutris solution seems to work great for me. Thanks for sharing. I have recently made the switch to Linux, but for Strinova I still had to dual boot into Windows up until now.
> 
> Only issue I really have is that the models in the menus look off. It's like they're missing proper lighting and the edges are also visible. This is also apparent in the character select screen before a match. The map in the background in the character select screen also loads partially. In the match itself, the models and the map look proper though. Running on everything on ultra settings.
> 
> Just wanted to share my experience but if anyone has an idea how to resolve it, that would be great. The game is perfectly playable otherwise.
> 
> ![Image](https://github.com/user-attachments/assets/2d85b89a-0ab7-458d-a87f-0b732e286cd7)
> Click for more screenshots
> 
> ![Image](https://github.com/user-attachments/assets/d86d4ce9-9197-4354-8627-2ad2a72a38d8)

</details>

**Comment by ktnk-dev** [comment]
@0x6A7232 https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2957287524

I don't think it's possible without building wine by yourself with gstreamer libs. BGM doesn't work because (in my case) wine installation from rpm repos doesn't have gstreamer included. I found it by executing these commands 
```bash
ldd "$(which wine)" | grep gst
```

`$(which wine)` can be replaced with a path to custom wine executable

**Comment by navewindre** [comment]
it seems a certain kernel version (>=6.14?) is required for the game to load properly. on 6.11.9 the game would hang on the gray transition screen after the login splash with 0% cpu usage, only rarely getting past it. updating to 6.15.2 fixed it. ntsync does not affect it.

**Comment by 0x6A7232** [comment]
> it seems a certain kernel version (>=6.14?) is required for the game to load properly. on 6.11.9 the game would hang on the gray transition screen after the login splash with 0% cpu usage, only rarely getting past it. updating to 6.15.2 fixed it. ntsync does not affect it.

6.14 is the kernel version that merged ntsync

**Comment by 0x6A7232** [comment]
> [@0x6A7232](https://github.com/0x6A7232) [#8262 (comment)](https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2957287524)
> 
> I don't think it's possible without building wine by yourself with gstreamer libs. BGM doesn't work because (in my case) wine installation from rpm repos doesn't have gstreamer included. I found it by executing these commands
> 
> ldd "$(which wine)" | grep gst
> 
> `$(which wine)` can be replaced with a path to custom wine executable

Just realized ... I was poking around installing various gstreamer plugins to try to get this working when I checked in <Strinova dir>/drive_c/Strinova/log/log_<datetime>.log and CTRL+F for music and found this repeated again and again

```
06-09 19:15:25.763: Debug/(0): music enter  

06-09 19:15:25.784: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.806: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.830: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.855: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.879: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.903: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.929: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.953: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.976: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:25.998: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.019: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.045: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.066: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.087: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.116: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.140: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.164: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.190: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.213: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.236: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.255: Debug/(0): music leave 

06-09 19:15:26.262: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.290: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.316: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.342: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.370: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.394: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.418: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.445: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.469: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.493: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.518: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.542: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.552: Debug/(0): music enter  

06-09 19:15:26.565: Warning/(0): DirectShowPlayerService::doRender: Unknown error 0x80040218.
06-09 19:15:26.581: Debug/(0): music leave 
```

Google seems to think installing K-Lite would help so maybe a WINE prefix issue (depends on codec that isn't included by default?

**Comment by ktnk-dev** [comment]
@0x6A7232 
I dont have same logs
<details>
<summary><code>Strinova/log/log_DATETIME.log</code></summary>
<img src="https://github.com/user-attachments/assets/cd007ee3-f469-4a60-948f-bec5e8d495e4">
<b>I dont play strinova right now because unstable network connection and huge packet loss</b>
</details>
<details>
<summary><code>Lutris logs</code> (spams entire logs with this)</summary>
<pre>(wine:779676): GStreamer-CRITICAL **: 15:35:48.642: gst_object_unref: assertion 'object != NULL' failed
(wine:779676): GStreamer-CRITICAL **: 15:35:48.643: gst_element_set_state: assertion 'GST_IS_ELEMENT (element)' failed
(wine:779676): GStreamer-CRITICAL **: 15:35:48.643: gst_object_unref: assertion '((GObject *) object)->ref_count > 0' failed
(wine:779676): GStreamer-CRITICAL **: 15:35:48.643: gst_element_set_bus: assertion 'GST_IS_ELEMENT (element)' failed</pre></details>

Next time ping me, thanks

**Comment by ktnk-dev** [comment]
@navewindre https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2964421697
I am on `6.14.5-300`, thats strange

**Comment by 0x6A7232** [comment]
> [@0x6A7232](https://github.com/0x6A7232) I dont have same logs
> `Strinova/log/log_DATETIME.log`
> `Lutris logs` (spams entire logs with this)
> 
> Next time ping me, thanks

I'm probably missing something but I have no way of checking *what* is missing.  Especially if you don't build WINE with gstreamer, then it relies on system gstreamer but IDK which ones it needs.

Is there anyone who has the standard WINE-staging (as of this writing, that's 10.10) installed that is using the Lutris install script for Strinova that has their main menu background music working?

If so, can you open you package manager and type in gstreamer and list the packages that are installed?  You may need to uncheck the equivalent of "Hide library and developer packages" to get the complete list.  

<details><summary>

I attached a screenshot of mine with which BGM isn't working.  Also I attached 2 screenshots of the gstreamer packages which I do *NOT* have installed.  In addition, I've also tried installing `gstreamer1.0-qt5, gstreamer1.0-qt6, and gstreamer1.0-python3-plugin-loader`, which did not fix anything so I uninstalled them.  In addition, I installed the 32 bit versions of these packages, which didn't fix the problem, so I uninstalled them and a massive list (100+) of dependencies that they brought with them: `gstreamer1.0-plugins-base:i386, gstreamer1.0-plugins-good:i386, gstreamer1.0-plugins-bad:i386, gstreamer1.0-plugins-ugly:i386, gstreamer1.0-libav:i386`

</summary>

![Image](https://github.com/user-attachments/assets/bf97ef12-1e9b-44d2-bd37-fab7df4bfd09)

![Image](https://github.com/user-attachments/assets/68037069-3e16-44e1-af80-f15d2a0ee7ef)

![Image](https://github.com/user-attachments/assets/6a7c52b1-7c4e-4407-ba26-2c89689726e1)

</details>

**Comment by 0x6A7232** [comment]
Ok, update, I finally got enough room expanded on my partition to fully install Strinova via Lutris (Steam defaults to library on another drive, but Lutris games are on the primary partition... TMI, moving on).

I actually selected the Strinova Steam installation location in the game launcher for the install, as it should have the exact same data.
This didn't work before because there wasn't enough space on the launcher drive for it to even consider it.  Now that there is, it proceeded.

I have successfully gotten to the main menu inside the game.  The main menu music works.  However, the intro movie only plays audio and subtitles, the image is frozen either on the first frame of the video, or on the placeholder image that displays before video start (not sure how stock Strinova normally works).

When playing the game tutorial, cutscenes are shown as black with subtitles and audio works.  Game character and directions render fine.

Interestingly, `lsof /dev/ntsync` still shows nothing.

**Comment by ktnk-dev** [comment]
@0x6A7232 https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2979062034

that's interesting. normally Strinova just plays video without background music in the launcher, and normally plays cutscenes and intro video without issues. but running a game properly still requires wine 10.4+, not proton (at least I can't run in Lutris with proton at all)

probably you will have some broken/missing textures with your method

**Comment by 0x6A7232** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2979246487

No no, I meant the menu music in the GAME works, not the launcher.  The launcher music is still broken (look at the top right of the launcher and you will see selection for music volume and music language, so it is supposed to work, and others have stated that it works for them).

**Comment by ktnk-dev** [comment]
### missing textures on latest game update
<details>
<summary>Screenshots</summary>
<img src="https://github.com/user-attachments/assets/ccef36f3-f878-4e55-96cd-b3a09fbf2163">
<img src="https://github.com/user-attachments/assets/b186db79-c0b7-403f-8279-1743d6f01b16">
</details>

<details>
<summary>Config</summary>
<img src="https://github.com/user-attachments/assets/13159fb4-2e66-4018-8dee-d7ff75c85e20">
</details>

nothing was changed in configs since last time i played strinova, idk why this happened 
tested: `wine 10.7 staging`, `wine-tkg 10.4 ntsync staging`, `wine-tkg 10.10 ntsync staging`

**Comment by kisak-valve** [comment]
Hello @ktnk-dev, your feedback is with a third party launcher using wine staging. Please note that this isn't useful feedback for a Proton dev to ponder. Please use another medium to troubleshoot the behavior of wine staging.

If this game needs something in wine staging that isn't in upstream wine, then the way forward is to identify exactly which patch set is needed, refine it upstream so that it is accepted into the main upstream wine repo. Then after that's done, come back here and make a request for the merged commit(s) to be cherry picked to Proton Experimental.

**Comment by ktnk-dev** [comment]
@kisak-valve https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2993574059
before game update game was perfectly playable (from middle of may up until now)

third party launcher are NOT required to run game (using it just to easily switch between wine/proton versions), you can run just using wine 10.7 staging from official repo (or any tkg 10.4+ staging, normal 10.7+ staging) 

missing textures issue happened right after game update, so none of previously reported wine version works right now

previously game required specific version: 10.7 staging, after that someone found that wine-tkg 10.4 staging ntsync also works perfectly. now both unplayable 

my feedback important for other members (like @aamaanaa, @0x6A7232, and others), who can help find solution for new game version and identify why it requires staging release

currently its impossible to even run game using upstream wine and any of proton versions

**Comment by ktnk-dev** [comment]
maybe you misunderstood `missing textures on latest update`, i meant game update, not proton

**Comment by kisak-valve** [comment]
What you're written is that `<not Proton>` and `<not Proton>` previously worked, and now has changed. Right now at this moment, that's irrelevant because you've drifted outside of the project which this issue tracker is meant for.

We're not trying to be a catch-all for the entire Linux gaming ecosystem. Use another issue tracker or forum closer to wine staging to troubleshoot the change in behavior outside of Proton.

**Comment by ktnk-dev** [comment]
@kisak-valve https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-2993591715
now i get it, thanks

since this is the only place where we discuss this, we will move to other issue tracker/platform in the future

**Comment by 0x6A7232** [comment]
> What you're written is that `<not Proton>` and `<not Proton>` previously worked, and now has changed. Right now at this moment, that's irrelevant because you've drifted outside of the project which this issue tracker is meant for.
> 
> We're not trying to be a catch-all for the entire Linux gaming ecosystem. Use another issue tracker or forum closer to wine staging to troubleshoot the change in behavior outside of Proton.

I see what you are saying.  Of note, though, is the fact that `<Proton>` doesn't work at all, and any insight into what makes the game work on systems that are similar to Proton may help.  However, I get that this isn't the proper channel for that.  So as it stands right now:

Strinova is unusable on Steam Proton.  Move on, close thread.

**Comment by ktnk-dev** [comment]
Strinova runs on Steam Deck for some reason, but not on regular PC with Proton

**Comment by 0x6A7232** [comment]
> Strinova runs on Steam Deck for some reason, but not on regular PC with Proton

I wonder if it would run on an Arch-based Linux like Manjaro (SteamOS is based on Arch), and if so, what is the difference that makes it work?

**Comment by ktnk-dev** [comment]
@0x6A7232 https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-3016633157
it runs on Arch (also on NVIDIA without graphical issues) [link to issue](https://github.com/aamaanaa/strinova-linux/issues/8)

**Comment by 0x6A7232** [comment]
A bit of an update: NTSYNC now enabled in GE-Proton with option `PROTON_USE_NTSYNC=1` as of v 10.9, but use 10.10 as that has 32-bit fixes.
BGM menu music now works!

With Strinova in main menu:

```
joshua@mx-Linux-KP9USMC:~
$ ls /dev/ntsync
/dev/ntsync
joshua@mx-Linux-KP9USMC:~
$ lsof /dev/ntsync
COMMAND      PID   USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
steam.exe 290556 joshua   14r   CHR 10,125      0t0   92 /dev/ntsync
wineserve 290558 joshua   14r   CHR 10,125      0t0   92 /dev/ntsync
services. 290562 joshua   12r   CHR 10,125      0t0   92 /dev/ntsync
winedevic 290565 joshua   12r   CHR 10,125      0t0   92 /dev/ntsync
svchost.e 290574 joshua   12r   CHR 10,125      0t0   92 /dev/ntsync
plugplay. 290582 joshua   12r   CHR 10,125      0t0   92 /dev/ntsync
winedevic 290590 joshua   12r   CHR 10,125      0t0   92 /dev/ntsync
explorer. 290606 joshua    7r   CHR 10,125      0t0   92 /dev/ntsync
rpcss.exe 290616 joshua   12r   CHR 10,125      0t0   92 /dev/ntsync
tabtip.ex 290626 joshua    7r   CHR 10,125      0t0   92 /dev/ntsync
xalia.exe 290628 joshua    7r   CHR 10,125      0t0   92 /dev/ntsync
Strinova. 290635 joshua   12r   CHR 10,125      0t0   92 /dev/ntsync
joshua@mx-Linux-KP9USMC:~
```

However, no matter if I set `Auto` to `0` and delete the `Debugger` value in `HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\AeDebug` or leave `Auto` at `1` and `Debugger` set to `winedbg --auto %ld %ld`, I get the following error:


> Anti-cheat system works error. Please close related applications that may affect anti-cheat modules, or check system abnormal events with anti-virus softwares. Reboot system and try to start the game process in administrator.
> 
> (13-131079-63)
> 

<img width="319" height="141" alt="Image" src="https://github.com/user-attachments/assets/93e313c0-6883-4368-84ee-ecc7e817c2e5" />

Side note: there is an additional AeDebug registry key set of `Auto` and `Debugger` on 64-bit Windows which I learned of in [this thread](https://stackoverflow.com/questions/1014345/how-do-i-stop-windbg-from-being-the-interactive-debugger#comment59745190_1014433) which may need to be cleared as it is present on my Strinova install.  It is located in `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows NT\CurrentVersion\AeDebug` -- I tried setting both sets to `Auto` `0` and blanking `Debugger` and it still has the same problem.

@ktnk-dev Is there another registry location that is setting off the anti-cheat?  Or perhaps an update broke it?

**Comment by ghost** [comment]
@0x6A7232 can u discuss LUTRIS specific issues in the repo i have made? look on my profile. there is some stuff, instructions and so on there.

btw as for ur eac error, i do not have it. some thing has broken on ur install.

**Comment by 0x6A7232** [comment]
> [@0x6A7232](https://github.com/0x6A7232) can u discuss LUTRIS specific issues in the repo i have made? look on my profile. there is some stuff, instructions and so on there.
> 
> btw as for ur eac error, i do not have it. some thing has broken on ur install.

My reply is not Lutris specific.  This is regarding getting it working on Steam, and the results in my reply were using Steam.  Although I suppose someone would gripe that I was using GloriousEggroll not Proton.  

Just re-tried today with the same results as above.  Let me try regular Proton Experimental... ok, nope, that has the same result.

**Comment by ktnk-dev** [comment]
@0x6A7232 https://github.com/ValveSoftware/Proton/issues/8262#issuecomment-3172236634
> My reply is not Lutris specific.

It isnt, but because there is no way to run Strinova using proton - any discussions here was stopped and moved to another repo. It happened because we dont use proton to run Strinova, dont know how to make it run on proton. I think this issue was created only to discuss proton and how to make [any game] run properly via proton. Consider opening issue on @aamaanaa repo
<details>
<summary>(click to reveal a link to repo)</summary>
<a href="https://github.com/aamaanaa/strinova-linux">https://github.com/aamaanaa/strinova-linux</a>
</details>

btw I played today and didnt have any issues

**Comment by ktnk-dev** [comment]
@kisak-valve update - works on 10.13~10.14 directly from steam (update: on 10.15 may instantly crash)

**Comment by 0PandaDEV** [comment]
For me, it runs fine, but there are some textures/shaders missing or bugging around.

Although I did use ProtonGE-10.16 which is not official.

OS: Arch Linux 6.16.8
GPU: RTX 3060 (580.82.09-5)

**Comment by blubskye** [comment]
Launches fine with Proton-GE 10.17 but fails with ACE anti-cheat to close things, even though all I have is my browser open on other monitors? OS: Fedora 42

**Comment by 0x6A7232** [comment]
Guys I guess unless you're using vanilla (what came with Steam) Proton we're not supposed to post here.  I guess either post in the aamaanaa repo or the GloriousEggroll repo?

**Comment by eroc1990** [comment]
> Guys I guess unless you're using vanilla (what came with Steam) Proton we're not supposed to post here. I guess either post in the aamaanaa repo or the GloriousEggroll repo?

Correct, because this repo is for Valve's Proton. Any other downstream versions are not relevant to this conversation and have their own repositories.

**Comment by alosslessdev** [comment]
If you have problems with background music you probably need to install gstreamer https://gstreamer.freedesktop.org/documentation/installing/on-linux.html?gi-language=c

**Comment by ghost** [comment]
> Guys I guess unless you're using vanilla (what came with Steam) Proton we're not supposed to post here. I guess either post in the aamaanaa repo or the GloriousEggroll repo?

steam proton issues = here
lutris / bottles issues = my repo
proton-ge, or any other non valve runtime = thier own repos

do not post here for lutris issues or any non steam / steam proton issues,  otherwise @kisak-valve will get angry ;P
